osama-breakout-room/Shirin Muataz/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="Style.css" />
+    <script src="https://kit.fontawesome.com/9016f6ba0a.js"></script>
+  </head>
+
+  <body>
+    <header>
+      <div class="wrapper">
```

Comment:
Awesome work on using the wrapper class to give a fixed width and centering your webpage content. 👍